cmd/wallet-js-sdk/test/specs/didcomm/connect.v2.spec.js

```diff
@@ -35,13 +35,15 @@ before(async function () {
       keyType: "ecdsap256ieee1363",
       keyAgreementType: "p256kw",
       enableDIDComm: true,
+      contextProviderURL: ["http://localhost:10096/agent-startup-contexts.json"]
```

Comment:
Are you sure we need it since we are downloading all contexts from network

Reply:
same comment as above, we have a context provider hosted on port 10096